tools/src/main/java/org/infinispan/tools/config/ConfigurationConverter.java

```diff
@@ -0,0 +1,59 @@
+package org.infinispan.tools.config;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import org.infinispan.Version;
+import org.infinispan.configuration.cache.Configuration;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.parsing.ConfigurationBuilderHolder;
+import org.infinispan.configuration.parsing.ParserRegistry;
+
+/**
+ * ConfigurationConverter.
+ *
+ * @author Tristan Tarrant
+ * @since 9.0
+ */
+public class ConfigurationConverter {
```

Comment:
This looks very useful? Can we mention this tool in our docs somewhere (please ignore me if we already did).